src/mrinufft/__init__.py

```diff
@@ -35,7 +35,7 @@
     display_3D_trajectory,
 )
 
-from .density import voronoi, cell_count, pipe
+from .density import voronoi, cell_count, pipe, get_density
```

Comment:
In documentation I recommend mentioning a bit about each including computation times